app/script/message/MentionEntity.js

```diff
@@ -43,10 +43,20 @@ z.message.MentionEntity = class MentionEntity {
     this.userId = userId;
     this.userEntity = ko.observable();
 
-    this.isSelfMentioned = ko.pureComputed(() => {
-      const isTypeUserId = this.type === z.cryptography.PROTO_MESSAGE_TYPE.MENTION_TYPE_USER_ID;
-      return isTypeUserId && this.userEntity() && this.userEntity().is_me;
-    });
+    // id of the user who will read the mention
+    this.readerId = undefined;
+  }
+
+  setReaderId(userId) {
```

Comment:
We refer to the reader as self in any case. So let's continue to do so here: `setSelfId`